install-browser.mjs

```diff
@@ -28,14 +28,23 @@
  *  - `node install-browser.mjs /tmp/cache`
  */
 
-import fs from 'fs/promises';
+import {readFile} from 'fs/promises';
+import {homedir} from 'os';
+import {resolve} from 'path';
 
 import {install, computeExecutablePath} from '@puppeteer/browsers';
 import actions from '@actions/core';
 
+const SHELL_ARG = '--shell';
+
 try {
-  const browserSpec = (await fs.readFile('.browser', 'utf-8')).trim();
-  const cacheDir = process.argv[2] || process.cwd();
+  const browserSpec = (await readFile('.browser', 'utf-8')).trim();
+
+  let cacheDir = resolve(homedir(), '.cache/chromium-bidi');
```

Comment:
Maybe add this comment:

```
XXX: Consider https://www.npmjs.com/package/cache-or-tmp-directory or similar for portability.
```

This is for https://github.com/GoogleChromeLabs/chromium-bidi/issues/876. Windows is not a fan of directories that start with a period AFAIK.

No need to change anything here, code is great as is.

Reply:
We should be OK about this one Git creates `.git` and that seems to work fine.